db/migrate/20170329211205_create_tag_groups.rb

```diff
@@ -0,0 +1,11 @@
+class CreateTagGroups < ActiveRecord::Migration
+  def change
+    create_table :tag_groups do |t|
+      t.integer :team_id, index: true
+      t.string :name, limit: 15
+      t.text :description
+      t.string :bg_color
```

Comment:
You can add a default color here, or define a range of colors in the model to select some randomly to suggest it to user.